### ISSUE TITLE
It is no longer possible to create calculated columns on mssql

### ISSUE THREAD

**Issue by nickshoe** [liquibase/liquibase#2283]
## Environment

**Liquibase Version**: 4.6.1

**Liquibase Integration & Version**: maven

**Liquibase Extension(s) & Version**: 

- **liquibase-hibernate5**: 4.6.1

**Database Vendor & Version**:

- Microsoft SQL Server Developer (64-bit) 15.0.2080.9

**Operating System Type & Version**:
- Windows 10 Pro (10.0)

## Description

Liquibase 4.6.1 (and lower?) seems to require a value for the `type` property for the column specified in a `addColumn` change, this prevents the possibility to create a computed column on mssql as the generated SQL statement leads to a syntax error

## Steps to reproduce

The following _changeset_ worked with Liquibase 4.3.5:

```xml
<changeSet id="20200123095600-1" author="me" dbms="mssql">
    <addColumn tableName="event">
        <column name="payload_id AS JSON_VALUE(payload,'$.id')"
                       type=""
                       computed="true">
        </column>
    </addColumn>
    <createIndex tableName="event" indexName="idx_payload_id">
        <column name="payload_id"/>
    </createIndex>
</changeSet>
```

But, 4.6.1 gives me the following error:

```
liquibase.exception.ValidationFailedException: Validation Failed:
     1 changes have validation failures
          columnType is empty, config/liquibase/changelog/20200123110200_create_index_entity_Event.xml::20200123095600-1::me
```

It complains that the `type` attribute of the column specified in the `addColumn` _change_ is empty. \
If I specify the correct type as follows...

```
        <column name="payload_id AS JSON_VALUE(payload,'$.id')"
                       type="nvarchar(4000)"
                       computed="true">
        </column>
```
...Liquibase compiles it to the following SQL statement, which leads to a syntax error on mssql:

```
[Failed SQL: (102) ALTER TABLE event ADD payload_id AS JSON_VALUE(payload,'$.id') nvarchar(4000)]
```

## Expected/Desired Behavior

I expect the pre-existing `addColumn` block for creating computed columns on mssql to be working even in 4.6.1, either without specifying the `type` property or by correctly specifying it and having the generated SQL statement to be correct.

**Comment by nickshoe** [comment]
Maybe, the correct way to add such a column is to use a `sql` block:

```xml
<changeSet id="20200123095600-1" author="blutec" dbms="mssql">
     <sql>
         ALTER TABLE event ADD payload_id AS JSON_VALUE(payload,'$.id')
    </sql>
    <createIndex tableName="event" indexName="idx_payload_id"
        <column name="payload_id"/>
    </createIndex>
</changeSet>
```

**Comment by kataggart** [comment]
@nickshoe thanks for submitting this issue -- we are reviewing it and will definitely get back to you.

Also, if you haven't had a chance to check out [https://forum.liquibase.org/](https://forum.liquibase.org/) make sure to swing by -- it's becoming a pretty solid place for questions, community help, etc.

**Comment by nickshoe** [comment]
@kataggart thanks. Please, be aware that the I've experienced this issue on Microsoft SQL Server (`mssql`).

**Comment by anjiB72** [comment]
Having the same issue since we upgraded to 4.5.0 (from 3.10.3) - The calculated column is in our initial changelog so wouldn't be possible to now update that changelog.

Validation fails on the file.

**Comment by anjiB72** [comment]
Currently having the same issue as recently upgraded from 13.10.3 to 4.5.0.   Unable to update the changelog as it is our initial log that has been released.    The type was not specified (type=''") as it is a computed column but it is now failing validation upon updateSQL goal being run in maven.

**Comment by yodzhubeiskyi** [comment]
This issue is fixed by [#2340](https://github.com/liquibase/liquibase/pull/2340)

**Comment by nickshoe** [comment]
Thanks @yodzhubeiskyi, I'll try it out.
